src/test/java/lambda/part3/exercise/Exercise1.java

```diff
@@ -19,11 +20,17 @@ public void mapEmployeesToLengthOfTheirFullNames() {
         List<Employee> employees = Example1.getEmployees();
         List<Integer> lengths = new ArrayList<>();
 
-        // TODO функция извлечения информации о человеке из объекта сотрудника personExtractor: Employee -> Person
-        // TODO функция извлечения полного имени из информации о человеке fullNameExtractor: Person -> String
-        // TODO функция извлечения длины из строки stringLengthExtractor: String -> Integer
-        // TODO функция извлечения длины полного имени из сотрудника fullNameLengthExtractor: Employee -> Integer
-        // TODO преобразование списка employees в lengths используя fullNameLengthExtractor
+        Function<Employee, Person> personExtractor = Employee::getPerson;
+        Function<Person, String> fullNameExtractor = Person::getFullName;
+        Function<String, Integer> stringLengthExtractor = String::length;
+        Function<Employee, Integer> fullNameLengthExtractor = e ->stringLengthExtractor.apply(
```

Comment:
you'd better use andThen() function  to make code more readable
  [ example: personExtractor.andThen(fullNameExtractor).andThen(stringLengthExtractor)  ]